pages/common/nano.md

```diff
@@ -1,28 +1,32 @@
 # nano
 
-> Simple, easy to use command-line text editor. An enhanced, free Pico clone.
+> Command-line text editor.
 > More information: <https://nano-editor.org>.
 
-- Open a new file in nano:
+- Start the editor:
 
 `nano`
 
-- Open a specific file:
+- Start the editor without loading startup configs:
 
-`nano {{path/to/file}}`
+`nano --ignorercfiles`
 
-- Open a specific file, positioning the cursor at the specified line and column:
+- Open specific files:
```

Comment:
```suggestion
- Open specific files, moving to the next file when closing the previous one:
```

This could maybe worded more succinctly however I feel it is necessary since some editors now obviously have multiple page editing at one time such as VSCode for example, and therefore some people maybe expecting a similar layout and will be confused.